src/main/webapp/usermap.jsp

```diff
@@ -15,7 +15,7 @@
     <h1 class="caption">Who is using TEAMMATES?</h1>
     <div id="contentHolder">
         <div id="container" style="position: relative; width: 800px; height: 500px; border: 1px solid #DEDEDE;"></div>
-        <p id="lastUpdate" class="lastUpdate">Last updated: 30 November 2016</p>
+        <p id="lastUpdate" class="lastUpdate">Last updated: <span id="lastUpdateDate" class="lastUpdate"></span></p>
```

Comment:
>Remove class="lastUpdate" on the class tag?

This what you said. So i thought it was `p` tag , i got confused when you said `class` tag. But i ll try not repeat the same mistakes in feature.